README.md

```diff
@@ -1,68 +1,34 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Welcome to my Portfolio
 
-## Available Scripts
+This project was made using React and Elastic UI as a learning practice.
 
-In the project directory, you can run:
+This is the latest version is deployed here: www.beatrizsanchez.dev
 
-### `npm start`
+![My Portfolio image](src/assets/images/myPortfolioEducation.png)
 
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Setup
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
+To run this project, install it locally using npm:
 
-### `npm test`
+```
+$ cd ../myPortfolio
```

Comment:
This is not needed... whoever clones you repo might pick a different directory.

Reply:
Removed